driver/src/main/com/mongodb/client/FindFluent.java

```diff
@@ -122,4 +122,13 @@
      * @return this
      */
     FindFluent<T> cursorType(CursorType cursorType);
+
```

Comment:
Only tangentially related, but what do you think about throwing a count() terminator on FindFluent?